cysync/src/designSystem/genericComponents/coinIcons.tsx

```diff
@@ -11,6 +11,7 @@ import generic from '../../../node_modules/cryptocurrency-icons/svg/color/generi
 import ltc from '../../../node_modules/cryptocurrency-icons/svg/color/ltc.svg';
 import btct from '../../assets/icons/btct.svg';
 import ethr from '../../assets/icons/ethr.svg';
+import matic from '../../assets/icons/matic.svg';
```

Comment:
We can  use existing library for coin icon
```suggestion
import matic from '../../../node_modules/cryptocurrency-icons/svg/color/matic.svg';
```

Reply:
is this safe to do? statically referencing the node_modules directory.